jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2628,9 +2628,40 @@ Jobs are run concurrently by default, so you must explicitly require any depende
 | `requires`
 | N
 | List
-| A list of jobs that must succeed for the job to start. Note: When jobs in the current workflow that are listed as dependencies are not executed (due to a filter function for example), their requirement as a dependency for other jobs will be ignored by the requires option. However, if all dependencies of a job are filtered, then that job will not be executed either.
+| A list of jobs that must succeed or attain a certain status for the job to start. Note: When jobs in the current workflow that are listed as dependencies are not executed (due to a filter function for example), their requirement as a dependency for other jobs will be ignored by the requires option. However, if all dependencies of a job are filtered, then that job will not be executed either.
 |===
 
+[.table.table-striped]
+[cols=3*, options="header", stripes=even]
+|===
+| Key | Type | Description
+
+| `job_name`
+| Job Name
+| An upstream job that must `succeed` for the job to start.
+
+| `job_name`
+| Map
+| A map of job name to a list of statuses (possible values are `success`, `failed` and `canceled`). The job starts if the upstream dependency reaches one of the statuses.
+|===
+
+[,yml]
+----
+workflows:
+  my-workflow:
+    jobs:
+      - build
+      - test:
+          requires:
+            - build
+      - deploy:
+          requires:
+            - build
+            - test:
```

Comment:
I suggest adding an example to show it is possible to define a single status on the same line as the required job name, e.g.
```
requires:
  - foo: canceled
```